plugin/cd-project.lua

```diff
@@ -8,12 +8,3 @@ if vim.g.loaded_cd_project == 1 then
 	return
 end
 vim.g.loaded_cd_project = 1
-
-require("cd-project").setup()
```

Comment:
If we remove this setup call in the plugin entrypoint, the user have to call the setup function when they add the plugin in their configuration.

Reply:
Isn't that desirable?
Currently it's running twice if you have a custom configuration?